src/analytics/events.ts

```diff
@@ -115,13 +116,24 @@ async function fetchEvents({
 
 // NOTE: This file is gitignored. If you change its name, update it also in gitignore.
 const cachedEventsFilePath = "wasp-analytics-cached-events.json";
+const cacheedEventsRemotePath = "wasp-analytics-cached-events.json";
```

Comment:
```suggestion
const cachedEventsRemotePath = "wasp-analytics-cached-events.json";
```

Will also require updating below.